mopidy/backend.py

```diff
@@ -228,6 +228,9 @@ def is_live(self, uri):
         """
         return False
 
+    def download_track(self):
```

Comment:
`def should_download(self, uri) ...` is probably better, as it allows the backend to decide per track instead of us locking this in as a per backend setting. And if it was a per backend setting having it as a method makes no sense.

Also please add documentation for this method, just use `is_live`'s as a starting point.